app/Services/Twilio.php

```diff
@@ -0,0 +1,164 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: ivan
+ * Date: 04.06.16
+ * Time: 15:24
+ */
+
+namespace App\Services;
+
+use App\Models\Countries;
+use App\Models\PhoneNumbers;
+use Services_Twilio;
+use Exception;
+use App\Exceptions\NoTwilioAvailableNumberException;
+use App\Exceptions\NoTwilioIncomingNumberException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class Twilio
+{
+
+    const THANKING_MESSAGE_DURATION = 120; //60 * 2 seconds
+
+    const DEFAULT_APPLICATION_SID = 'AP6452316e22874c8e9f4c3425832b0b28';
+
+    const SHORT_CONVERSATION_MESSAGE = 'Thank you very much';
+    const LONG_CONVERSATION_MESSAGE = 'We were glad to here you';
+
+    const PHONE_NUMBER_TYPE_LOCAL = 'Local';
+    const PHONE_NUMBER_TYPE_TOLL_FREE = 'TollFree';
+    const PHONE_NUMBER_TYPE_MOBILE = 'Mobile';
+    
+    /**
+     * @var Services_Twilio
+     */
+    private $client;
+
+    /**
+     * Twilio constructor.
```

Comment:
It was done to remove phpstorm highliting of undefined property variables and methods